### PR TITLE
Changeable lift speeds

### PR DESCRIPTION
**Pull request by csakip**
Switched to pyphotonfile 0.2.1
Lift speeds can be changed.
Exposure time can be float.

I don't have Qt Creator so ui_mainwindow.py was edited directly.

### CODE REVIEW

**Comment by fookatchu** [comment]
Thanks for the work! I am currently quite busy with other projects, but I plan to do a new release sometime down the road once I fixed some issues with pyphotonfile. Until then I won't be able to merge the pull request. I will most likely pick some of your stuff and do a refactor so that there are not two 'SL1Reader' classes, so don't ne discouraged if I don't merge the whole pull request.

**Comment by cab404** [comment]
@csakip Hello! Can you please PR this to https://github.com/cab404/SL1ToPhoton as well? I will maintain a fork until @fookatchu finds time to do so here)

**Comment by cab404** [comment]
This PR is unfocused, and merging it actually breaks output files. Please, refine your changes to the scope of PR name, @csakip )

File: SL1_to_Photon.py
Comment (cab404) [inline SL1_to_Photon.py:81]: All of those settings do not fit this PR, and require explanation.

File: SL1_to_Photon_gui.py
Comment (cab404) [inline SL1_to_Photon_gui.py:143]: As well as those

**Review by cab404 — CHANGES_REQUESTED** [review]

File: SL1_to_Photon.py
Comment (cab404) [reply SL1_to_Photon.py:81]: Ooof, sorry, didn't update pyphotonfile prior to review — everything is clearer now)

Comment (cab404) [reply SL1_to_Photon.py:81]: Merged that into my fork

**Pull request closed without merging**
